index.html

```diff
@@ -72,6 +76,191 @@ <h1>
+              <p>2015</p>
+            </div>
+            <p class="work-child">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <ul class="frameworks-list work-child">
+              <li class="framework">html</li>
+              <li class="framework">css</li>
+              <li class="framework">javaScript</li>
+            </ul>
+            <button type="button" class="project-btn work-child">See Project</button>
+          </div>
+        </div>
+        <div class="wraper second">
+          <div class="image">
+            <img alt="work2" src="./assets/imgs/Snapshoot2.svg" />
+          </div>
+          <div class="work-info">
+            <h2 class="work-child">Multi-Post Stories</h2>
+            <div class="description-wraper work-child">
+              <h3 class="project-name">FACEBOOK</h3>
+              <img alt="dot" src="./assets/imgs/Counter.png" />
+              <p>Full Stack Dev</p>
+              <img alt="dot" src="./assets/imgs/Counter.png" />
+              <p>2015</p>
+            </div>
+            <p class="work-text work-child">
+              Experimental content creation feature that allows users to add to
+              an existing story over the course of a day without spamming their
+              friends.
+            </p>
+            <ul class="frameworks-list work-child">
+              <li class="framework">html</li>
+              <li class="framework">Ruby on rails</li>
+              <li class="framework">css</li>
+              <li class="framework">javaScript</li>
+            </ul>
+            <button type="button" class="project-btn work-child">See Project</button>
+          </div>
+        </div>
+        <div class="wraper">
+          <div class="image">
+            <img alt="work3" src="./assets/imgs/Snapshoot3.png" />
+          </div>
+          <div class="work-info">
+            <h2 class="work-child">Facebook 360</h2>
+            <div class="description-wraper work-child">
+              <h3 class="project-name">FACEBOOK</h3>
+              <img alt="dot" src="./assets/imgs/Counter.png" />
+              <p>Full Stack Dev</p>
+              <img alt="dot" src="./assets/imgs/Counter.png" />
+              <p>2015</p>
+            </div>
+            <p class="work-text work-child">
+              Exploring the future of media in Facebook's first Virtual Reality
+              app; a place to discover and enjoy 360 photos and videos on Gear
+              VR.
+            </p>
+            <ul class="frameworks-list work-child">
+              <li class="framework">html</li>
+              <li class="framework">Ruby on rails</li>
+              <li class="framework">css</li>
+              <li class="framework">javaScript</li>
+            </ul>
+            <button type="button" class="project-btn work-child">See Project</button>
```

Comment:
- [ ] I think you have not added some functionalities to the buttons `(Hover, Enabled, Pressed, Disabled)`. Following the Figma design, you're supposed to apply these functionalities. Kindly check the screenshot below and fix all the necessary buttons. 
![image](https://user-images.githubusercontent.com/62849872/143662166-72d5c7a5-cc5c-4c9b-8514-2e695f269366.png)